app/src/main/java/com/simpletrack/model/Task.kt

```diff
@@ -0,0 +1,34 @@
+import java.util.Date
+
+class Task() {
+    var start: Date? = null
+        get
+        private set
+    var stop: Date? = null
+        get
+        private set
+
+    fun startTime() {
+        if (start == null) {
+            start = Date()
+        }
+    }
+
+    // returns stopped time in milliseconds
```

Comment:
Note: You should use Docstring for comments like that. See https://kotlinlang.org/docs/kotlin-doc.html#generate-the-documentation